Makefile

```diff
@@ -2,6 +2,22 @@
 
 include Makefile.include
 
+TARGET ?= _bash
+
+env-up-ex:                    ## Start experimental devcontainer.
```

Comment:
I created new tasks so that we can experiment to a while and decide if we want to use this approach